src/containers/Profiles/EditProfile.tsx

```diff
@@ -68,23 +86,43 @@ const EditProfile = () => {
   if (isLoading || isFetching) return <LoadingOverlay inline />;
 
   return (
-    <div className={styles.user}>
-      <div className={styles.leftColumn}>
-        <div className={styles.panel}>
-          <div className={profileStyles.avatar}>
-            <h2>Howdy{`${fullName && ','} ${fullName}`}</h2>
-            <img src={profileDetails?.avatar_url} />
+    <div className={classNames(styles.user, contained && profileStyles.contained)}>
+      {!contained && (
+        <div className={styles.leftColumn}>
+          <div className={styles.panel}>
+            <div className={profileStyles.avatar}>
+              <h2>
+                {t('profile.greeting')}
+                {`${fullName && ','} ${fullName}`}
```

Comment:
It might be better to include the name placeholder in the translations because we can't guarantee that the sentence structure is the same for all languages.

```suggestion
                {fullName ? t('profile.greeting_with_name', { fullName }) : t('profile.greeting')}
```

**translations:**
```json
{
  "profile": {
    "greeting_with_name": "Greetings, {{fullName}}!"
  }
}
```

Reply:
https://github.com/jwplayer/ott-web-app/pull/339/commits/9a982546bdf0e0a7f14ae46c73ccc9237327acd1